components/FavoriteButton/FavoriteButton.tsx

```diff
@@ -0,0 +1,52 @@
+// Libraries
+import React, { useState, useEffect, useMemo } from "react";
```

Comment:
```suggestion
import React, { useState, useEffect } from "react";
```
if not needed remove